pkg/vault/test/foundry/unit/VaultCommonModifiers.t.sol

```diff
@@ -1,5 +1,9 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
 pragma solidity ^0.8.24;
 
+import "forge-std/Test.sol";
```

Comment:
Seems like unnecessary import

Reply:
It is, but I understand it as a pattern in test files. Not sure if we should stick to this pattern or not, but it makes it easy to console.log stuff, for example, when we need to debug something.